pages/index.js

```diff
@@ -59,7 +59,7 @@ export default function HomePage() {
           {isFound === false && (
             <>
               <p>added new word</p>
-              <Link href="/words">show entry {``}</Link>
+              <Link href="/words">show entry {`>>`}</Link>
```

Comment:
instead of this you might consider using an icon or unix character here.

Reply:
Good idea, I changed it.